_data/tutorials.yaml

```diff
@@ -268,3 +268,15 @@ cogrouping-streams:
     ksql: disabled
     kstreams: enabled
     kafka: disabled
+
+console-consumer-produer-basic:
+  title: "Console Producer and Consume Basics, no (de)serializers"
+  meta-description: "Produce and consume your first message with Kafka"
+  slug: "/kafka-console-consumer-producer-basics"
+  problem: "you are brand new to Kafka and you want to experiment by producing and consuming a message quickly"
+  introduction: "So you are excited to get started with Kafka and youd like to produce and consume some basic messages and you want to do so quickly.  In this tutorial we'll show you how to produce and consume messages from the command line with no code! Note this tutorial doesn't cover using (de)serializers. Using (de)serializers with the console consumer and producer are covered in separate tutorials"
```

Comment:
@ybyzek does the updated title and introduction satisfy the comments from https://github.com/confluentinc/kafka-tutorials/pull/339#discussion_r418656996?